.github/.bazelrc

```diff
@@ -0,0 +1,12 @@
+# Bazel configuration that will be copied to /etc/bazel.bazelrc.
```

Comment:
What is this file? Why is it under `.github/`? How is it different from the root `.bazelrc`?

Reply:
The root `.bazelrc` is for all environments and users. This file is only used under CI because it has specific commands that only CI should use (as mentioned below). If you follow the link, you'll see that there's precedence given to Bazel RC files in the loading phase:

- Top-level (/etc/bazel.bazelrc)
- Workspace-level (.bazelrc)
- User-level (.bazelrc.user - loaded by .bazelrc at the end of the file)

Reply:
Let's move it to the root and rename it `.bazelrc-ci` or something like that so it will be discoverable next to the root one.

Reply:
When you say this, do you mean instead of copying to `/etc/bazel.bazelrc`, or just that it shouldn't be hidden in `.github` to start?

Reply:
Shouldn't be hidden